packager/media/event/combined_muxer_listener.h

```diff
@@ -42,7 +42,8 @@ class CombinedMuxerListener : public MuxerListener {
   void OnNewSegment(const std::string& file_name,
                     int64_t start_time,
                     int64_t duration,
-                    uint64_t segment_file_size) override;
+                    uint64_t segment_file_size,
+                    int64_t segment_index) override;
```

Comment:
Prefer `segment_number` so it matches with the file_name. What do you think? A number of files will need to be updated.

Reply:
@kqyang Do you want to change segment_index in struct SegmentInfo, SegmentEventInfo, and kSegmentIndex in unit tests to kSegmentNumber as well? Or just the methods with file_name parameter in it?

Reply:
I think we should. There should be as few unnecessary "+1" or "-1" as possible. (If I have overlooked anything, let me know)